### DISCUSSION TITLE
Is it possible to use with wordpress (woocommerce) as a backend?

### DISCUSSION THREAD

**Discussion by amr3k** [itswadesh/svelte-commerce#24] [Q&A]
Also thanks for making this promising project.

**Comment by itswadesh** [comment]
Thank you so much.

Yes it is possible if you have ReST api from woocommerce.

**Reply by bcombes** [reply]
@itswadesh the woocommerce integration seems to be broken atm, I fixes some minor errors in `src/lib/services/woocommerce/product-service.ts` and `src/lib/services/woocommerce/store-service.ts` (fixes reproduced in the screen shots) but it doesn't seem to be loading products still...
<img width="965" alt="image" src="https://user-images.githubusercontent.com/1664437/236639705-a7f20e94-52d6-4797-97d7-a21b200c1afd.png">
I would love to take a stab and at getting this running, but a bit new to the tools and frameworks used.
